### DISCUSSION TITLE
We plan to build a Chinese mirror site.

### DISCUSSION THREAD

**Discussion by codingcn** [nuxt/nuxt#16208] [Ideas]
Hello, I am a nuxt user from China and have been using the framework since nuxt2. Thank you very much to the author for providing such a great framework. 👍 

## We plan to build a Chinese mirror site

I plan to build a Chinese version of nuxt website with some friends. The starting point is as follows:

- nuxt2 previously provided multi-language support, but not Chinese, we are not clear about nuxt3 in the future.

- Due to the 'Great Firewall' in China, developers usually need to use 'vpn' to access foreign servers (such as github, google, etc.) stably. Not every developer in China has a vpn tool.

- Due to policy restrictions, foreign entities who want to conduct website business in China need to ICP filing, etc., which is a very tedious process. [ICP Recordal](https://www.amazonaws.cn/en/support/icp/)

- At the same time, I also see that there have been related issues before, but it seems to be progressing slowly. https://github.com/nuxt/nuxt/discussions/16054

## What we are doing ?

- Build a Chinese Nuxt website based on 'nuxt/framework/docs' and' nuxt-website-theme '(translation progress: 80% complete)

- Use the domain name 'https://nuxt.com.cn', with Nuxt's official domain name only more than a '.cn 'suffix.

- Replace 'Roobert PRO' with 'Inter font' (not started)

- File the domain name and use the Chinese mainland CDN to speed up the site (not started)

- Translation progress is synchronized with official nuxt/framework (ongoing...)

- We will not claim to be the official website of nuxt, we just want to provide a Chinese mirror site.

So far we've done most of the work, but we'd also like to know what the officials have to say about this ❤️

visit https://nuxt.com.cn

@pi0 @danielroe

**Comment by nazarepiedady** [comment]
@codingcn could you share your source code?
I would like to extract the source structure to build a Portuguese mirror site.
